modules/ossm-control-plane-deploy.adoc

```diff
@@ -84,21 +79,45 @@ Follow this procedure to deploy the {ProductName} control plane the command line
 ----
 $ oc login https://{HOSTNAME}:6443
 ----
-
++
 . Create a project named `istio-system`.
 +
 [source,terminal]
 ----
 $ oc new-project istio-system
 ----
-
-. Create a `ServiceMeshControlPlane` file named `istio-installation.yaml` using the example found in "Customize the {ProductName} installation". You can customize the values as needed to match your use case.  For production deployments you _must_ change the default Jaeger template.
-
-. Run the following command to deploy the control plane:
++
+. Create a `ServiceMeshControlPlane` file named `istio-installation.yaml` using the following example in "Customize the {ProductName} installation". For production deployments you _must_ change the default Jaeger template.
```

Comment:
It isn't clear how the user should change the Jaeger template. Also, what is meant by "template"? Is it just the `spec.addons.jaeger` section in the SMCP?